src/Mod/Path/PathScripts/PathToolController.py

```diff
@@ -131,9 +131,9 @@ def propertyEnumerations(self, dataType="data"):
         # Enumeration lists for App::PropertyEnumeration properties
         enums = {
             "SpindleDir": [
-                (translate("Path_ToolController", "None"), "None"),
                 (translate("Path_ToolController", "Forward"), "Forward"),
                 (translate("Path_ToolController", "Reverse"), "Reverse"),
+                (translate("Path_ToolController", "None"), "None"),
```

Comment:
I'm not familiar with the enum/translation logic - does the order of values matter? For "normal" enums it does, does it here?

Reply:
Yes.  That's why I moved 'None' to the end.  So that 'Forward' will be the default value on creation.